<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionWidget</class>
 <widget class="QWidget" name="ConnectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>270</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>270</height>
      </size>
     </property>
     <property name="title">
      <string>Server status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="serverComboBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Select service</string>
          </property>
          <item>
           <property name="text">
            <string>IGS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WING</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LGS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CyberORO</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tygem</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eWeiQi</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tom</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="changeServerButton">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>25</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Change Server</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="openCheckBox">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Others can invite you to a match</string>
        </property>
        <property name="text">
         <string>Open for playing</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="lookingCheckBox">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Looking for a game</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="quietCheckBox">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Don't receive list updates from server</string>
        </property>
        <property name="text">
         <string>No lists update</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QToolButton" name="seekToolButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Seek a match</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="application.qrc">
            <normaloff>:/ressources/pics/not_seeking.png</normaloff>:/ressources/pics/not_seeking.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="handicapComboBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Set the handicap range you want for seeking games</string>
          </property>
          <item>
           <property name="text">
            <string>Even</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Within 2 stones</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Within 5 stones</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Anyone</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Same or stronger</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Same or weaker</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="roomsLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Room</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="roomComboBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="channelsLabel">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="channelComboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Lists</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="0">
        <widget class="QComboBox" name="filterRank1ComboBox">
         <property name="toolTip">
          <string>Only show players within these ranks</string>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9p-1p</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NR</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QPushButton" name="refreshGamesButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Refresh games list</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="view-refresh"/>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="filterFriendsCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Only show friends</string>
         </property>
         <property name="text">
          <string>Friends</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QCheckBox" name="filterWatchesCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Only show games played by watched players</string>
         </property>
         <property name="text">
          <string>Watch</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="9">
        <widget class="FilteredView" name="gamesView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="filterOpenCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Only show players available for a match</string>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="editFriendsWatchesButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Change the friends/watches/blocks lists</string>
         </property>
         <property name="text">
          <string>Edit Friends/Watch</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="9">
        <widget class="FilteredView" name="playerView">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="createRoomButton">
         <property name="text">
          <string>Create Room</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="filterRank2ComboBox">
         <property name="toolTip">
          <string>Only show players within these ranks</string>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9p-1p</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1d</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NR</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QPushButton" name="refreshPlayersButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Refresh players list</string>
         </property>
         <property name="icon">
          <iconset theme="view-refresh"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Console</string>
      </attribute>
      <layout class="QVBoxLayout" name="_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QTextEdit" name="MultiLineEdit2">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="commandLineComboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="talkTabs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabShouts">
      <attribute name="title">
       <string>Shouts</string>
      </attribute>
      <layout class="QGridLayout" name="_5">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextEdit" name="shoutTE">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMessages">
      <attribute name="title">
       <string>Messages</string>
      </attribute>
      <layout class="QGridLayout" name="_6">
       <property name="margin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QTextEdit" name="msgTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonErase">
         <property name="text">
          <string>Erase messages</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="applications-internet" resource="application.qrc">
     <normaloff>:/ressources/pics/connect_no2.png</normaloff>:/ressources/pics/connect_no2.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <zorder>groupBox_3</zorder>
  <zorder>talkTabs</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilteredView</class>
   <extends>QTableView</extends>
   <header>listviews.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="application.qrc"/>
 </resources>
 <connections/>
</ui>
